pkg/cmd/release/deploy/deploy.go

```diff
@@ -562,7 +581,7 @@ func AskQuestions(octopus *octopusApiClient.Client, stdout io.Writer, asker ques
 
 		if !isExcludedStepsSpecified {
 			// select steps to exclude
-			deploymentProcess, err := deployments.GetDeploymentProcess(octopus, space.ID, selectedRelease.ProjectDeploymentProcessSnapshotID)
+			deploymentProcess, err := deployments.GetDeploymentProcessByID(octopus, space.ID, selectedRelease.ProjectDeploymentProcessSnapshotID)
```

Comment:
This was required as a part of an earlier change to the go-octopusdeploy project that hadn't been included in the CLI yet.